proto3-suite.cabal

```diff
@@ -10,7 +10,17 @@ build-type:          Simple
 cabal-version:       >=1.10
 data-files:          test-files/*.bin tests/encode.sh tests/decode.sh
 
+Flag dhall
+  Description:   Turn on Dhall interpret and inject codegen
+  Default:       False
```

Comment:
You'll probably also want to add `Manual: True` so that `cabal` doesn't silently disable the `dhall` flag if it can't find a build plan

Reply:
See: https://www.haskell.org/cabal/users-guide/developing-packages.html#pkg-field-flag-manual